frontend/src/reducer.ts

```diff
@@ -169,38 +169,34 @@ export function appStateReducer(state: CoveyAppState, update: CoveyAppUpdate): C
     case 'messageReceived':
       switch (update.message.type) {
         case MessageType.TownMessage:
-          nextState.townMessageChain = nextState.townMessageChain.addMessage(update.message);
+          nextState.townMessageChain.addMessage(update.message);
```

Comment:
Is there a reason you took out the variable assignment?

Reply:
i dont think you need to do variable assignment since the addMessageFunction is being used in other cases without it (see the proximity case)

Reply:
I took it out because this reducer allows you to directly mutate the state, you don't really need to reassign. that's a bad way to setup a reducer imo but it saves us some code